cmd/clock_test.go

```diff
@@ -0,0 +1,30 @@
+// +build test
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/jmhodges/clock"
+)
+
+// Clock functions similarly to clock.Default(), but the returned value can be
+// changed using the FAKECLOCK environment variable in test environments.
+//
+// The FAKECLOCK env var is in the time.UnixDate format, returned by `date -d`.
+func Clock() clock.Clock {
+	if tgt := os.Getenv("FAKECLOCK"); tgt != "" {
+		targetTime, err := time.Parse(tgt, time.UnixDate)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "cmd.Clock: bad format for FAKECLOCK: %v\n", err)
+			os.Exit(2)
+		}
+
+		cl := clock.NewFake()
+		cl.Set(targetTime)
```

Comment:
Add a `GetAuditLogger().Notice()` that we are changing the internal clock and what it is being set to.